### PR TITLE
Try to modify external gflags to solve CI compilation

### PR DESCRIPTION
**Pull request by zhhsplendid**
```
[11:51:07]	CMakeFiles/jit_kernel_benchmark.dir/benchmark.cc.o: In function `_GLOBAL__sub_I_benchmark.cc':
[11:51:07]	benchmark.cc:(.text.startup+0x576): undefined reference to `google::FlagRegisterer::FlagRegisterer<std::string>(char const*, char const*, char const*, std::string*, std::string*)'
[11:51:07]	../../platform/dynload/libdynamic_loader.a(dynamic_loader.cc.o): In function `_GLOBAL__sub_I_dynamic_loader.cc':
[11:51:07]	dynamic_loader.cc:(.text.startup+0x9c): undefined reference to `google::FlagRegisterer::FlagRegisterer<std::string>(char const*, char const*, char const*, std::string*, std::string*)'
[11:51:07]	dynamic_loader.cc:(.text.startup+0x13a): undefined reference to `google::FlagRegisterer::FlagRegisterer<std::string>(char const*, char const*, char const*, std::string*, std::string*)'
[11:51:07]	dynamic_loader.cc:(.text.startup+0x1d8): undefined reference to `google::FlagRegisterer::FlagRegisterer<std::string>(char const*, char const*, char const*, std::string*, std::string*)'
[11:51:07]	dynamic_loader.cc:(.text.startup+0x276): undefined reference to `google::FlagRegisterer::FlagRegisterer<std::string>(char const*, char const*, char const*, std::string*, std::string*)'
[11:51:07]	../../platform/dynload/libdynamic_loader.a(dynamic_loader.cc.o):dynamic_loader.cc:(.text.startup+0x314): more undefined references to `google::FlagRegisterer::FlagRegisterer<std::string>(char const*, char const*, char const*, std::string*, std::string*)' follow
[11:51:07]	collect2: error: ld returned 1 exit status
[11:51:07]	paddle/fluid/operators/jit/CMakeFiles/jit_kernel_benchmark.dir/build.make:155: recipe for target 'paddle/fluid/operators/jit/jit_kernel_benchmark' failed
[11:51:07]	make[2]: *** [paddle/fluid/operators/jit/jit_kernel_benchmark] Error 1
[11:51:07]	CMakeFiles/Makefile2:69574: recipe for target 'paddle/fluid/operators/jit/CMakeFiles/jit_kernel_benchmark.dir/all' failed
[11:51:07]	make[1]: *** [paddle/fluid/operators/jit/CMakeFiles/jit_kernel_benchmark.dir/all] Error 2
[11:51:07]	make[1]: *** Waiting for unfinished jobs....
```
http://ci.paddlepaddle.org/viewLog.html?buildId=132680&buildTypeId=Paddle_PrCi&tab=buildLog&_focus=2071

### CODE REVIEW

**Comment by zhhsplendid** [comment]
Maybe wrong due to this line of log:

```
[06:50:23]W:	 [Step 1/1] make[3]: warning: jobserver unavailable: using -j1.  Add '+' to parent make rule.
```

Changed based on reference:
https://stackoverflow.com/questions/33171335/jobserver-unavailable-when-building-external-projects-with-cmake

File: cmake/external/gflags.cmake [hunk 1]
```diff
@@ -49,7 +50,37 @@ ExternalProject_Add(
...
+  )
+ELSE(WIN32)
+  ExternalProject_Add(
+    extern_gflags
+    ${EXTERNAL_PROJECT_LOG_ARGS}
+    GIT_REPOSITORY  "https://github.com/gflags/gflags.git"
+    GIT_TAG         77592648e3f3be87d6c7123eb81cbad75f9aef5a
+    PREFIX          ${GFLAGS_SOURCES_DIR}
+    BUILD_COMMAND   $(MAKE) --silent
+    INSTALL_COMMAND $(MAKE) install
```
Comment (luotao1) [inline cmake/external/gflags.cmake]: 可以更简洁一点么？
```
if NOT(WIN32)
   set(BUILD_COMMAND $(MAKE) --silent)
   set(INSTALL_COMMAND $(MAKE) install)
endif()
```
```
BUILD_COMMAND   ${BUILD_COMMAND}
INSTALL_COMMAND ${INSTALL_COMMAND}
```

Comment (zhhsplendid) [reply cmake/external/gflags.cmake]: Done

**Review by luotao1 — APPROVED** [review]
LGTM! Nice job! Thanks very much!

**Pull request merged**